### PR TITLE
icon size

### PR DESCRIPTION
**Pull request by alexsosim**

### CODE REVIEW

File: lib/src/style/QlementineStyle.cpp [hunk 1]
```diff
@@ -1492,8 +1492,8 @@ void QlementineStyle::drawControl(ControlElement ce, const QStyleOption* opt, QP
           const auto colorize = QlementineStyle::isAutoIconColorEnabled(w);
           const auto& colorizedPixmap = colorize ? getColorizedPixmap(pixmap, fgColor) : pixmap;
           const auto pixmapPixelRatio = colorizedPixmap.devicePixelRatio();
-          const auto pixmapW = pixmapPixelRatio != 0 ? (int) ((qreal) colorizedPixmap.width() / pixmapPixelRatio) : 0;
```
Comment (oclero) [inline lib/src/style/QlementineStyle.cpp:1495]: Why did you remove the multiplication factor? I guess it was necessary to get the correct size, wasn't it?

Comment (alexsosim) [reply lib/src/style/QlementineStyle.cpp:1495]: yes, let me close this PR and create another one
